lang/en/texts/contribute-to-open-food-facts-by-playing.html

```diff
@@ -0,0 +1,74 @@
+
+<h3 style="background-color:#00d400;padding:1%;color:#fff;font-weight:bold;">1. The question</h3>
+<p>To answer the question, just click on one of the 3 options (Yes/No/Skip) and the game will automatically move onto the next product.</p>
+
+<p>At the top of the screen are 4 buttons (filters) allowing you to select the type of data you wish to complete:</p>
+<ul>
+<li><strong>label</strong>: relating to the "label" field (eg. certification, product origin, etc.)</li>
+<li><strong>category</strong>: relating to the "category" field (eg drinks, sweets, etc.)</li>
+<li><strong>brand</strong>: relating to the brand of the product (eg Sodébo, Charal, Alter Eco, etc.)</li>
+<li><strong>product_weight</strong>: relating to the weight of the product (eg 100 g, 1L, etc.)</li>
+</ul>
+<p>Below we find:</p>
+<ol>
+<li>A search field, allowing you to specify which brand, label, category you want to work on,</li>
+<li>The question to be answered in English. In English, this gives: "Does this product correspond to the indicated value?"</li>
+<li>Product image,</li>
+<li>3 clickable buttons to answer the question.</li>
+</ol>
+
+<p><strong>When you are not 100% sure of your answer you must answer "Skip"!</strong>Above all, do not answer "Yes" or "No" if you are not sure of your choice.
+Another person will take care of qualifying this data later! This game is above all collaborative and aims to contribute efficiently and qualitatively to Open Food Facts data.</p>
+
+<h3 style="background-color:#00d400;padding:1%;color:#fff;font-weight:bold;">2. Access information</h3>
+<p>If the answer is not visible on the product image, this part of the screen allows you to view the product information. You can click on:</p>
+<ul>
+<li>Images to zoom,</li>
+<li>The "view" button: provides access to the product file,</li>
+<li>The "edit" button: gives access to the file's editing interface.</li>
+</ul>
+<p>This last option is useful when the product requires further modification.</p>
+<h3 style="background-color:#00d400;padding:1%;color:#fff;font-weight:bold;">3. Annotations</h3>
+<p>On the far right of the screen (or at the very bottom of the screen if you are on mobile), there are annotations that let you know:</p>
+<ul>
+<li>The number of products remaining to be treated (varies depending on the filters applied),</li>
+<li>The number of annotated products,</li>
+<li>A history of the latest actions taken during your session.</li>
+</ul>
+
+</div>
+
+<p>If you have any questions or comments, please don't hesitate to contact us on the Slack or on <a href="https://www.facebook.com/groups/openfoodfacts.fr/">the Facebook group</a>.</p>
+
+<h2>Thank you!</h2>
+
+<p>Open Food Facts is a non-profit project that could not exist without your help. Data on the foods we eat have many uses and can
+change the lives of many people. This is why all the data we collect is freely accessible to everyone, in open data, so that we can build a common good that benefits everyone.</p>
```

Comment:
```suggestion
<p>Open Food Facts is a non-profit project that could not exist without your help. Data on the foods we eat have many uses and can change the lives of many people. This is why all the data we collect is freely accessible to everyone, in open data, so that we can build a common good that benefits everyone.</p>
```